modules/network-observability-configuring-CPU-memory-limit.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+
+// * configuring-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-config-CPU-memory-limit{context}"]
+
+= Increasing Network Observability operator memory limits
+You can increase memory limits for the Network Observability operator by patching the CSV:
+
+.Procedure
+
+. Run the following command to patch the CSV:
++
+[source,terminal]
+----
+$ oc -n netobserv patch csv network-observability-operator.v1.0.0 --type='json' -p='[{"op": "replace", "path":"/spec/install/spec/deployments/0/spec/template/spec/containers/0/resources/limits/memory", value: "1Gi"}]'
+clusterserviceversion.operators.coreos.com/network-observability-operator.v1.0.0 patched
```

Comment:
Line 18 is not part of the command.  It is the output you get from the command.

Reply:
Addressed in Servesha's troubleshooting PR: https://github.com/openshift/openshift-docs/pull/53713/files